.travis.yml

```diff
@@ -28,8 +26,14 @@ script:
 matrix:
   include:
     - rvm: 2.6.1
-      gemfile: gemfiles/rails_5.gemfile
+      gemfile: gemfiles/rails_6.gemfile
       env: TEST_SUITE="rubocop lib/ spec/"
+  exclude:
+    - rvm: 2.3.6
+      gemfile: gemfiles/rails_6.gemfile
```

Comment:
@mgomes : do you know how to exclude some test in travis ? rails 6 require ruby >= 2.5

Reply:
Sorry I found the mistake.